README.md

```diff
@@ -1,10 +1,17 @@
 # Overview
 
-This repo provides example JSON body sections to enable and disable the CIS Benchmarks for GCP in Lacework when calling the PATCH method on *https://lacework-tenant.lacework.net/api/v1/external/recommendations/gcp* API Endpoint
+This repo provides example JSON body sections to enable and disable the CIS Benchmarks for Azure in Lacework when calling the PATCH method on *https://lacework-tenant.lacework.net/api/v1/external/recommendations/azure* API Endpoint via the Lacework CLI
 
 # Script
-This repo also includes an example script that can be used to perform the config, with a requirement of having curl and the Lacework CLI installed.  Usage is:
+## azure-cis-config.sh
+This repo also includes an example script that can be used to perform the config, with a requirement of having curl and the Lacework CLI installed and configured.  Usage is:
 
-`gcp-cis-config.sh [disable_cis_10|disable_cis_12|enable_cis_12] [lacework-tenant]`
+`azure-cis-config.sh [disable_cis_10|disable_cis_12|enable_cis_12] [lacework-tenant]`
```

Comment:
The disable/enable cis 1.2 needs to be updated to 1.3.1
Also the enable cis 1.0 option is missing?